mimesis/builtins/pl.py

```diff
@@ -0,0 +1,91 @@
+"""Specific data provider for Poland (pl)."""
+
+from typing import Optional
+
+from mimesis.builtins.base import BaseSpecProvider
+from mimesis.enums import Gender
+from mimesis.providers import Datetime
+from mimesis.typing import DateTime
+
+__all__ = ['PolandSpecProvider']
+
+
+class PolandSpecProvider(BaseSpecProvider):
+    """Class that provides special data for Poland (pl)."""
+
+    class Meta:
+        """The name of the provider."""
+
+        name = 'poland_provider'
+
+    def nip(self) -> str:
+        """Generate random valid 10-digit NIP.
+
+        :return: valid 10-digit NIP
+        """
+        nip_digits = [int(d) for d in str(self.random.randint(101, 998))]
+        for _ in range(6):
+            nip_digits.append(self.random.randint(0, 9))
+        nip_coefficients = (6, 5, 7, 2, 3, 4, 5, 6, 7)
+        sum_v = sum(
```

Comment:
This looks not really good. Can we simplify it?

Reply:
Did you mean `for _ in range(6):` part? If yes, then I can create one list with 3 digits (tax office code) then concatenate it with list of other 6 digits. If that would be simply enough?